sql/sql_expression.cc

```diff
@@ -2960,6 +2960,20 @@ printJsonTyped(const SelectExpression * val,
     else context.writeStringUtf8(val->surface);
 }
 
+/*****************************************************************************/
+/* JOIN QUALIFICATION                                                        */
+/*****************************************************************************/
+
+DEFINE_ENUM_DESCRIPTION(JoinQualification);
+
+JoinQualificationDescription::
+JoinQualificationDescription()
+{
+    addValue("JOIN_INNER", JOIN_INNER, "Inner join");
```

Comment:
I was more concerned with the associated string.  If I am not mistaken, someone would have to specify "JOIN_INNER" in a given JSON payload which, I feel, is not natural.